documents/operating.md

```diff
@@ -41,13 +41,90 @@ SECTION 3: 🤔🤔🤔 Projects 🤔🤔🤔
 ---------------------------------------------------------------------
 SECTION 4: 💵💵💵 MONEY 💵💵💵
 ---------------------------------------------------------------------
-- [ ] Capital contributions
-- [ ] Allocating resources to projects
-- [ ] Distributing profits
-- [ ] Compensation
-- [ ] Expenses
-- [ ] Taxes
-- [ ] Company canoe
+- [x] Capital contributions
+- [x] Allocating resources to projects
+- [x] Distributing profits
+- [x] Compensation
+- [x] Expenses
+- [x] Taxes
+- [x] Company canoe
+
+
+### Capital Contributions
+Each Corporate Overlord will contribute an $300 initial capital contribution
+to BIFFUD within three months of achieving Corporate Overlord status.
```

Comment:
I would like us to stick with $300 -- it is a clean number and the $$ will be put to use over the years (or given back, if everything falls apart)